python/cudf/cudf/_lib/aggregation.pyx

```diff
@@ -196,7 +195,7 @@ class Aggregation:
         # Handling UDF type
         nb_type = numpy_support.from_dtype(kwargs['dtype'])
         type_signature = (nb_type[:],)
-        ptx_code, output_dtype = cudautils.compile_udf(op, type_signature)
+        ptx_code, output_dtype = cudf.utils.cudautils.compile_udf(op, type_signature)
```

Comment:
Config setting needs to be in place before `numba.cuda` is imported, which otherwise happens here during cython import